src/ciphersuite/mod.rs

```diff
@@ -176,13 +183,15 @@ impl ExporterSecret {
     }
 }
 
-#[derive(Debug, PartialEq)]
+#[derive(Debug)]
```

Comment:
Let's not derive `Debug` for keys and secrets but only when people really know what they are doing. I filed #247 for this.